app/assets/javascripts/cascade/level/rotator.js

```diff
@@ -31,7 +31,7 @@ $(function () {
         var beforeSlideChange = function(slider) {
             if ( !isSliderMasthead ) { return; }
 
-            $header.fadeOut();
```

Comment:
Ok. That makes sense. Yes, this is better. I would add a comment explaining this. Future dev might be tempted to add back fadeOut/fadeIn thinking it does the same thing (which it almost does).